kubernetes/istio_details_service.go

```diff
@@ -5,11 +5,10 @@ import (
 	"strings"
 	"sync"
 
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/types"
-
 	"github.com/kiali/kiali/config"
 	"github.com/kiali/kiali/log"
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
wrong import order, the previous ones were good

Reply:
Approached!

Reply:
Thanks! BTW, I'm not sure about jetbrain's goland if it's what you use, but with vscode you can have this automatically done when a file is saved (in "settings" > "Go configuration" > "Format tool" set to "goreturns".)